### PR TITLE
Fixed PHPUnit suite and removed deprecated calls.

### PR DESCRIPTION
**Pull request by klaussilveira**

### CODE REVIEW

**Comment by westy92** [comment]
Why not use PHPUnit 6?

**Comment by klaussilveira** [comment]
There are many things that the library needs:

- [ ] Upgrading to PHPUnit 6
- [ ] Upgrading to PSR-4
- [ ] Upgrading to Guzzle 6
- [ ] Adhering to PSR-1 and PSR-2

However, i'm trying to keep most pull requests small and focused. Baby steps.

File: .gitignore [hunk 1]
```diff
@@ -1,12 +1,9 @@
-coverage
```
Comment (sndsgd) [inline .gitignore]: it looks like these have been sorted alphabetically. if we're going to change the order here, can we update to be directories in alphabetical order, followed by files in alphabetical order?

File: composer.json [hunk 2]
```diff
@@ -16,24 +11,21 @@
...
     "eher/oauth": "1.0.*",
-    "guzzle/guzzle": ">=3.1.0,<4"
+    "guzzlehttp/guzzle": ">=3.1.0,<4"
   },
-
   "require-dev": {
-    "phpunit/phpunit": "*"
+    "phpunit/phpunit": "~5.0"
+  },
+  "config": {
```
Comment (sndsgd) [inline composer.json]: this appears to be unnecessary

File: test/RequestHandlerTest.php [hunk 3]
```diff
@@ -4,26 +4,23 @@ class RequestHandlerTest extends \PHPUnit_Framework_TestCase
 {
     protected function setUp()
     {
-        $headers = $this->getMock('Guzzle\Http\Message\Header\HeaderCollection',
-                                  array('setHeaders', 'getHeaders', 'toArray'),
+        $headers = PHPUnit_Framework_TestCase::createMock('Guzzle\Http\Message\Header\HeaderCollection',
```
Comment (sndsgd) [inline test/RequestHandlerTest.php]: Can you explain your reasoning for this change? I realize that phpunit defines these static methods, but all the docs seem to use the instance aliases, so I believe they're the preferred pattern. At the time of writing it looks like https://phpunit.de/ is down, but docs for the most recent version of the library illustrate the pattern: https://phpunit.readthedocs.io/en/8.0/test-doubles.html#stubs.

**Comment by yahoocla** [comment]
Thank you for submitting this pull request, however I do not see a valid CLA on file for you. Before we can merge this request please visit https://yahoocla.herokuapp.com/ and agree to the terms. Thanks! :smile:

**Comment by yahoocla** [comment]
Thank you for submitting this pull request, however I do not see a valid CLA on file for you. Before we can merge this request please visit https://yahoocla.herokuapp.com/ and agree to the terms. Thanks! :smile:

**Comment by yahoocla** [comment]
Thank you for submitting this pull request, however I do not see a valid CLA on file for you. Before we can merge this request please visit https://yahoocla.herokuapp.com/ and agree to the terms. Thanks! :smile:

**Pull request closed without merging**